### ISSUE TITLE
Rbush 2.x compatibility

### ISSUE THREAD

**Issue by panayotoff** [MazeMap/Leaflet.LayerGroup.Collision#4]
Hi;

I just installed Rbush latest version ( 2.0.1 ) as a dependency, and it's not working out of the box. Adding support for 2.x or bundling with the old version would be great.

Best,
Chris

**Comment by meersonv** [comment]
Yes, it seems that the library is not compatible with last rbush version
